test/real_cases_periodic_test.rb

```diff
@@ -222,6 +222,7 @@ def test_treatment_site
 
     def test_route_initialisation
       vrp = TestHelper.load_vrp(self)
+      vrp.configuration.resolution.solver = false
```

Comment:
This change doesn't satisfies me :/
I'm currently considering to add a new parameter to allow "vrp resolution" after scheduling

Reply:
Why don't you like this modification?

I mean I don't like it either but my problem with it is the fact that if you turn the solver off `vrp.configuration.resolution.solver = false` the solver is not going to be called even for the sub-problems themselves.

Reply:
I think if we are considering a new parameter, a parameter to limit the "size" of problems that we launch the solver would be better no? We already have a solver=false/true which we can base our decision on.